desktop-widgets/command_divelist.cpp

```diff
@@ -176,14 +182,19 @@ void UndoAddDive::undo()
 	MainWindow::instance()->refreshDisplay();
 }
 
-UndoDeleteDive::UndoDeleteDive(const QVector<struct dive*> &divesToDeleteIn) : divesToDelete(divesToDeleteIn.toStdVector())
+DeleteDive::DeleteDive(const QVector<struct dive*> &divesToDeleteIn) : divesToDelete(divesToDeleteIn.toStdVector())
 {
 	setText(divesToDelete.size() > 1 ?
-			gettextFromC::tr("delete %1 dives").arg(divesToDelete.size()) :
-			gettextFromC::tr("delete dive"));
+			tr("delete %1 dives").arg(divesToDelete.size()) :
+			tr("delete dive"));
```

Comment:
see my previous comment on plurals - I realize that this was wrong in the existing code as well. Shame on us.